INDEX.md

```diff
@@ -11,6 +11,12 @@ the Flutter team works to keep this list up to date, there are plenty of others
 created by the community, such as
 [Awesome Flutter](https://github.com/Solido/awesome-flutter) from @Solido.
 
+***Please don't submit pull requests directly updating this file. While we're
+always happy to learn of new samples from the community, we need
+to keep this file small. There are plenty of user-maintained indices (like
+[Awesome Flutter](https://github.com/Solido/awesome-flutter)) that are meant to
+be exhaustive, and those are great places to submit your own stuff to.***
```

Comment:
to submit your own stuff to => for submitting your own work

Dangling prepositions, no!

Reply:
Guilty as charged...